### DISCUSSION TITLE
Ore Tweaker 4 - Is under development

### DISCUSSION THREAD

**Discussion by EwyBoy** [EwyBoy/OreTweaker#80] [Announcements]
**So a quick heads up here!**

I just started working on *Ore Tweaker 4* for Minecraft 1.19
It will most likely be nothing close to like the previous versions of Ore Tweaker due to the changes Mojang / Forge has made to the game.
I have no ETA for a release, and not sure how the final product will look like. I am doing the best I can with the tools I have available to me.

I have not made enough progress to claim the new system is going to be better or worse than the old format. 
It will most likely still have a file system based interface like the `data` folder in Ore Tweaker 2 / 3 and still use `.json` files.
Except from that everything else will be quite different. I will share more info when I progress in my research into this new system and will ask the community for feedback in important decisions.  

Thank you for the patients!
-Ewy

**Comment by EwyBoy** [comment]
![image](https://user-images.githubusercontent.com/5883716/189996216-29bf352c-4d14-4d24-a93a-96f9d0edafde.png)

**Comment by EwyBoy** [comment]
Work on this will resume in after new year. Once I get back from Christmas vacation, somewhere in the start of 2023 🎅🎇

**Comment by corporalrivalle** [comment]
Hey, any updates?

**Comment by Bebobobo** [comment]
if you know anyone that is able to help. https://github.com/EwyBoy/OreTweaker/issues/96

**Comment by EwyBoy** [comment]
https://github.com/EwyBoy/OreTweaker/discussions/106
